user/notifications.md

```diff
@@ -71,7 +71,7 @@ By default, a build email is sent to the committer and the author, but only if
 they have access to the repository the commit was pushed to. This prevents forks
 active on Travis CI from notifying the upstream repository's owners when they're
 pushing any upstream changes to their fork. It also prevents build notifications
-from going to folks not registered on Travis CI.
+from going to forks not registered on Travis CI.
```

Comment:
Absolutely. I don't even know what it means to register a fork. You can register a few **folks**, though.